saveSvgAsPng.js

```diff
@@ -151,8 +272,6 @@
         clone.setAttribute('transform', clone.getAttribute('transform').replace(/translate\(.*?\)/, ''));
 
         var svg = document.createElementNS('http://www.w3.org/2000/svg','svg')
-        svg.appendChild(clone)
```

Comment:
That logic is only hit if you pass a node within an SVG rather than the SVG itself. It probably shouldn't be removed for the changes you've made.